.circleci/config.yml

```diff
@@ -9,12 +9,12 @@ version: 2.1
 parameters:
   ubuntu-2004-docker-image:
     type: string
-    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2004-24
-    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:b8b645fa7ab40d55f2d16eac295d16ca01ec51d32be7d668ae6eaecd47dbd763"
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2004-25
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:b3f321fb2d8e7a41ca9328672061c1840e5cd3fb5be503aa158d1c508deacf0a"
   ubuntu-2204-docker-image:
     type: string
-    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204-9
-    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:80247de9655b1f39afd4ac22b14266bc9b9a0d64b283ae8fb9cb5b8250e4e77d"
+    # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204-10
+    default: "solbuildpackpusher/solidity-buildpack-deps@sha256:ba2d878c26d681a7d6c1922258b47c2e5dd61d0e46ab4c6a4862b473e61997b6"
```

Comment:
I really liked how @r0qs linked to the PRs hashes came from in #15422. It helps a lot when reviewing. We should always be doing that when the update is spread over more than the usual 2 PRs and you can get lost in them. It would have helped to at least link the PR as a dependency in the description.

In this case:
- `ubuntu2204-10`: https://github.com/ethereum/solidity/pull/15321#issuecomment-2291228820
- `ubuntu2004-25`: https://github.com/ethereum/solidity/pull/15321#issuecomment-2291247595
- `ubuntu2204.clang-9`: https://github.com/ethereum/solidity/pull/15321#issuecomment-2291230249